examples/preact/src/App.js

```diff
@@ -3,12 +3,23 @@ import { hot } from 'react-hot-loader'
 import Counter from './Counter'
 import { Internal } from './Internal'
 
+const Inner = ({ children }) => <div>{children}</div>
+
+const indirect = {
+  element: () => (
```

Comment:
This element is "invisible" to RHL "v3". Ie for RHL+Preact+(no-VDOM)
The internal state always will be lost.